app/v1/controllers/requestParsers/ListLossClaimsParser.scala

```diff
@@ -32,6 +32,17 @@ class ListLossClaimsParser @Inject()(val validator: ListLossClaimsValidator) ext
       incomeSourceType <- typeOfLoss.toIncomeSourceType
     } yield incomeSourceType
 
-    ListLossClaimsRequest(Nino(data.nino), taxYear.map(DesTaxYear.fromMtd), incomeSourceType, data.selfEmploymentId)
+    val claimType = for {
```

Comment:
Can this be shortened? Something like `data.claimType.map(ClaimType.parser)` (just looking at how we've done optional enums in student loans)

Reply:
Or `data.claimType.flatMap(ClaimType.parser.lift)` (going by how it's done here - honestly I don't know what the difference is)